Themes updates for 1.17 (#634)
MicrosoftDocs/terminal: TerminalDocs/customize-settings/profile-appearance.md

This is the name of the color scheme used in the profile. Color schemes are defined in the `schemes` object. More detailed information can be found on the [Color schemes page](./color-schemes.md).

> This property can alternatively accept a pair of color scheme names, as follows:

Reviewer: ```suggestion
In addition to a single color scheme name, this property can accept a pair of color scheme names as follows:
```

```json
"colorScheme":
{
    "light": "One Half Light",
    "dark": "One Half Dark",
},
```

> When specified in this manner, the Terminal will automatically switch between the two given color schemes, depending on the theme of the application. The Terminal will obey the [`theme.applicationTheme`](./themes#application-theme) property of the Terminal's selected theme. If that `applicationTheme` is set to `system`, then this will instead use the color scheme matching the theme as set by the OS.

Reviewer: ```suggestion
When specified in this manner, the Terminal will automatically switch between the two given color schemes depending on the theme of the application. The Terminal will follow the [`theme.applicationTheme`](./themes#application-theme) property of the Terminal's selected theme. If that `applicationTheme` is set to `system`, then this will instead use the color scheme matching the OS theme.
```

> [!IMPORTANT]
> Unblurred opacity (`"useAcrylic": false`) only works on Windows 11.

> [!IMPORTANT]

> > When Mica is enabled in the [theme settings](./themes#mica), Mica will appear underneath the Terminal contents as well, when the `opacity` of the Terminal is set to a value <100.

Reviewer: ```suggestion
> When Mica is enabled in the [theme settings](./themes#mica), Mica will appear underneath the Terminal contents when the `opacity` of the Terminal is set to a value <100.
```
